Tests/FluentAssertions.Specs/Primitives/TimeOnlyAssertionSpecs.cs

```diff
@@ -414,6 +414,20 @@ public void When_a_time_is_close_to_an_earlier_time_by_one_tick_it_should_fail()
             act.Should().Throw<XunitException>();
         }
 
+        [Fact]
+        public void A_precision_of_maximum_time_span_does_not_overflow()
+        {
+            // Arrange
+            TimeOnly time = new TimeOnly(23, 59, 0);
+            TimeOnly nearbyTime = new TimeOnly(0, 1, 0);
+
+            // Act
+            Act act = () => time.Should().NotBeCloseTo(nearbyTime, TimeSpan.MaxValue);
+
+            // Assert
+            act.Should().Throw<XunitException>();
```

Comment:
This line is intentional, because a `TimeOnly` (which has 24h) couldn't ever be close to a maximum time span.

Reply:
@jnyrup Is this test necessary at all? Because this test does not "test" anything.. (is never failing). Maybe we should remove it?

Reply:
Doesn't it exercise the `(precision == TimeSpan.MaxValue)` check?

Reply:
Yes. What I meant was: This could not fail, because the maximum time span is something around 10000 days. 

But if you do not have this check it does not fail the test.. 

> Looks good!
> I have no further comments on the functionality/implementation.
> Thanks for taking this through the iteration of changes necessary to this result.

No problem :)

we all love good clean code.. That is a necessary "evil" 😉

Reply:
> But if you do not have this check it does not fail the test..

Ah... I somehow thought `AddTicks(this TimeSpan ts, int ticks)` was implemented as `ts + ticks.Ticks()` which for `TimeSpan.MaxValue + 1.Ticks()` would use `TimeSpan.Add` and throw an `OverflowException`.
https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs,234
but instead we would do an unchecked overflow on long addition.

I'm beginning to think we should just copy and adjust `TimeOnly.IsBetween` to check for the closed interval instead.
With that implementation I would be fine with removing this test.

Reply:
To be honest: I think the first approach is less "magic" if you know what I mean. And the lesser change :D